virtualizarr/backend.py

```diff
@@ -133,9 +137,38 @@ def open_virtual_dataset(
         raise NotImplementedError()
 
     # if filetype is user defined, convert to FileType
+
     if filetype is not None:
         filetype = FileType(filetype)
 
+    if filetype == FileType.kerchunk_parquet:
+        from fsspec.implementations.reference import LazyReferenceMapper
+
+        from virtualizarr.readers.kerchunk import dataset_from_kerchunk_refs
```

Comment:
This is a change for a later PR but I think we want to tidy up these `if` statements to call a consistent API of some kind that's defined by each reader.